source/_docs/guides/drupal8-commandline.md

```diff
@@ -0,0 +1,376 @@
+The tasks covered in this guide can all be done in the browser, without using the command line.
+For instance, seeing a list of all of your sites on Pantheon is easy in the browser.
+It is the first thing you see when signing into the Dashboard.
+
+![A list of websites in the Pantheon dashboard](/source/docs/assets/images/drupal8-commandline--dashboard-sites.png)
+
+You can see the same list on the command line by running:
+
+`terminus site:list`
+
+In essence, Terminus is a way to do on the command line everything you can do in Pantheon's browser-based dashboard.
+Once you are comfortable with Terminus, you may find it faster to use than the browser.
+Terminus also opens the door to writing scripts that combine multiple Terminus commands to accommodate whatever workflow needs you have.
+For more information about Terminus itself, see our [Terminus Manual](/docs/terminus/).
+
+## Installing and authenticating with Terminus
+
+The most reliable way to install Terminus is with our dedicated installer:
+
+`curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar install`
+
+Once Terminus is installed, you can sign in with a [machine token generated in the Dashboard](https://dashboard.pantheon.io/machine-token/create).
+
+`terminus auth:login --machine-token=‹machine-token›`
+
+If you have trouble with these steps, [see our full installation documentation](https://pantheon.io/docs/terminus/install/).
+
+## Create your site with Dev, Test, and Live environments
+
+#### See your options for creating new sites
+
+Pantheon offers a few choices for spinning up new sites.
+You can see them all with.
+
+`terminus upstream:list`
+
+The one we want is `Drupal 8`.
+
+`terminus upstream:list | grep "Drupal 8"`
+
+That command shows us the unique ID for our Drupal 8 source repository.
+We will copy that ID (8a129104-9d37-4082-aaf8-e6f31154644e) into the next command for creating a new site.
+
+#### Creating the Drupal 8 site.
+
+In this command you will need to pick both a machine name and a label for your site.
+In my case, I'm picking `steve-site-d8` as my site's machine name and "Steve's Site D8" as the label.
+Finally, I'm adding that unique ID for the Drupal 8 source.
+
+So the command I am running is:
+
+`terminus site:create steve-site-d8 "My Site D8" 8a129104-9d37-4082-aaf8-e6f31154644e`
+
+If you are copy and pasting these examples, you will need to replace `steve-site-d8` in each one.
+
+There's an additional option you can put on this command for organization ID `--org`.
+If you are walking through this guide as part of an in person training or if you work as part of a team that uses Pantheon, you might want to associate this site with your organization.
+To get your organization's ID, run `terminus org:list`.
+So your command might look like
+
+`terminus site:create steve-site-d8 "My Site D8" 8a129104-9d37-4082-aaf8-e6f31154644e --org=123456-abcd-1234--abcd-1234567890`
```

Comment:
Org names can be used here too, e.g.:

`terminus site:create steve-site-d8 "My Site D8" "Drupal 8" "Pantheon Open Source Team"`